src/hooks/lastBlock/queries/useLastBlockQuery.tsx

```diff
@@ -0,0 +1,11 @@
+import { QueryHookOptions, useQuery } from '@apollo/client';
+import { loader } from 'graphql.macro';
+import { LastBlock } from '../../../generated/graphql';
+
+export const GET_LAST_BLOCK = loader('../graphql/GetLastBlock.query.graphql');
+export interface GetLastBlockQueryResponse {
+  lastBlock?: LastBlock;
+}
+
+export const useLastBlockQuery = (options?: QueryHookOptions) =>
```

Comment:
this query should be configured to use a cache-only fetch policy, we don't want it to try and hit a non existent local resolver